app/scripts/metamask-controller.js

```diff
@@ -1722,7 +1722,7 @@ export default class MetamaskController extends EventEmitter {
 
     delete connections[id]
 
-    if (Object.keys(connections.length === 0)) {
+    if (Object.keys(connections.length) === 0) {
```

Comment:
I think the paren is still in the wrong spot :sweat_smile: 

```suggestion
    if (Object.keys(connections).length === 0) {
```

Reply:
😱

Reply:
0779825

😑

Reply:
The third time's the charm hopefully?